chunky/src/java/se/llbit/chunky/ui/controller/ChunkyFxController.java

```diff
@@ -210,7 +212,24 @@ public GUIRenderListener(ChunkyFxController renderControls) {
     }
 
     @Override public void setSamplesPerSecond(int sps) {
-      this.sps = sps;
+      // Rolling average of SPS
```

Comment:
Isn't this method already called with the averages `sps`?  Shouldn't this be calculated from the spp and the time delta one step earlier?

Reply:
Whoops - missed that
I thought we didn't calculate the average at all